fighters/mario/src/opff.rs

```diff
@@ -213,6 +213,12 @@ pub unsafe fn moveset(fighter: &mut L2CFighterCommon, boma: &mut BattleObjectMod
     noknok_timer(fighter, boma, id);
     noknok_reset(fighter, id, status_kind);
     noknok_training(fighter, id, status_kind);
+
+    if fighter.is_button_on(Buttons::AppealHi) {
```

Comment:
hey actually why is this here?

Reply:
Accident left in for when I was testing. Has been resolved.